content/projects.md

```diff
@@ -1,21 +1,157 @@
 # Projects
 
+This document describes the organization and workflow for Optakt projects, and acts as a centralized table of contents of project documentation.
```

Comment:
Perhaps use "acts as a centralized table of contents *for* project documentation?"